installscripts/cookbooks/jenkins/files/default/jazz-installer-vars.json

```diff
@@ -88,21 +88,21 @@
               "API": {
                   "active": true,
                   "options": [
-                      {"label": "AWS API Gateway", "value": "aws_apigateway"},
-                      {"label": "GCP APIGEE", "value": "gcp_apigee"}
+                      {"label": "AWS API Gateway", "value": "aws_apigateway", "default": true},
+                      {"label": "GCP APIGEE", "value": "gcp_apigee", "default": false}
                   ]
               },
               "WEBSITE": {
                   "active": true,
                   "options": [
-                      {"label": "AWS S3", "value": "aws_s3"},
-                      {"label": "AWS Cloudfront", "value": "aws_cloudfront"}
+                      {"label": "AWS S3", "value": "aws_s3", "default": false, "disabled": true},
```

Comment:
Changes are good, let's target these for next release instead of targeting `apigee` release?